templates/data/spark/support.html

```diff
@@ -171,9 +181,14 @@ <h3 class="p-equal-height-row__item p-heading--5">Backed by an SLA</h3>
           </div>
           <div class="p-equal-height-row__col">
             <div class="p-equal-height-row__item p-image-container is-highlighted u-hide--small">
-              <img src="https://assets.ubuntu.com/v1/da8f6c14-firefighting%20support.png"
-                   alt=""
-                   class="p-image-container__image" />
+              {{ image(url="https://assets.ubuntu.com/v1/da8f6c14-firefighting%20support.png",
+                            alt="",
+                            width="568",
+                            height="853",
+                            hi_def=True,
+                            attrs={"class": "p-image-container__image"},
+                            loading="lazy") | safe
+              }}
 
             </div>
             <hr class="p-rule--highlight u-hide--medium u-hide--small" />
```

Comment:
Not related to your changes, but looks like there's a bug in this logo:
<img width="825" height="728" alt="image" src="https://github.com/user-attachments/assets/b16c22b4-9e60-4fa6-b445-733661307f31" />

Won't let me leave the suggestion on the relevant line but it's just missing the container ratio

Reply:
Thanks for spotting.